packages/components/src/components/atoms/form/TextFieldTagsMultiple.stories.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { useState } from 'react'
+import { ComponentMeta, ComponentStory } from '@storybook/react'
+
+import TextFieldTagsComponentMultiple from './TextFieldTagsMultiple'
+import {
+  ILimitations,
+  IOptionsTags,
+  ITransformedLimitations,
+} from '@elastic-suite/gally-admin-shared'
+
+const textOperatorOptions: IOptionsTags[] = [
```

Comment:
Pas besoin de typage spécifique ici, réutilise le typage `IOptions` qui existe déjà:
```suggestion
const textOperatorOptions: IOptions = [
```